src/powerflow_method.jl

```diff
@@ -0,0 +1,414 @@
+"""Abstract supertype for all variations of Newton-Raphson."""
+abstract type AbstractNewtonRaphsonMethod end
+"""Basic, no-frills Newton-Raphson."""
+struct SimpleMethod <: AbstractNewtonRaphsonMethod end
```

Comment:
```suggestion
struct SimpleNRMethod <: AbstractNewtonRaphsonMethod end
```
I know the information is available in the type tree but it seems more readable to put it in the type name too. Same for the siblings